### ISSUE TITLE
Accessibility -  Visual Impairment Function

### ISSUE THREAD

**Issue by jiraslama** [alan-turing-institute/AutSPACEs#317]
A wireframe design and justification that showcases the option to alter the contrast, to include a series of varied contrasts. For those with strong visual impairments a black background option with yellow text will be part of this.
